test/replicaset-luatest/replicaset_3_test.lua

```diff
@@ -247,6 +248,7 @@ test_group.test_locate_master_when_no_conn_object = function(g)
 end
 
 test_group.test_named_replicaset = function(g)
+    t.run_only_if(vutil.feature.persistent_names)
```

Comment:
How did it work in the previous commits without this check? You are using `identification_mode` here already in older commits, but didn't have the `run_only_if` before.